rpm/pacemaker.spec.in

```diff
@@ -306,6 +306,9 @@ BuildRequires: help2man
 BuildRequires: ncurses-devel
 BuildRequires: pam-devel
 
+# Required for "make check"
+BuildRequires: libcmocka-devel
+
```

Comment:
also need to add this to INSTALL.md (maybe as an optional dependency?)

make sure cmocka is available at least in debian and suse (if suse uses a different name, the spec will need to deal with that). If they aren't, that's a big downside

Reply:
It's available under the same name in SuSE and as libcmocka-dev in Debian.  I'll add it to INSTALL.md.  It's an optional dependency as long as your build system doesn't automatically invoke "make check", which using the spec file does.